src/Console/Application.php

```diff
@@ -188,11 +193,21 @@ private function buildDynamicDependencies(InputInterface $input)
                 $configLocation = \pathinfo($infectionConfigFile, PATHINFO_DIRNAME);
                 $json = file_get_contents($infectionConfigFile);
             } catch (\Exception $e) {
+                $infectionConfigFile = null;
                 $json = '{}';
                 $configLocation = getcwd();
             }
 
-            return new InfectionConfig(json_decode($json), $c['filesystem'], $configLocation);
+            $config = json_decode($json);
+
+            if (null === $config && 0 !== json_last_error()) {
+                throw new \RuntimeException(sprintf(
```

Comment:
We need to create our own exception based on the RuntimeException. As @theofidry usually tells - everything in the PHP can throw RuntimeException